### DISCUSSION TITLE
fix block-jumping bug/glitch

### DISCUSSION THREAD

**Discussion by Tizooo** [GriefPrevention/GriefPrevention#1484] [Ideas]
i know its not a rela bug/glitch but if a player places a block unterneath himself while jumping he can get up walls etc. with it worldguard somehow deals with it by teleporting the player back to his last position and i think it would be nice to see this small fix in griefprevention

**Comment by Jikoo** [comment]
GP doesn't do anticheat functions. That is not a simple or small feature.
